### ISSUE TITLE
Chatmanager not working properly with factions

### ISSUE THREAD

**Issue by kernitus** [PEXPlugins/PermissionsEx#592]
I had issues with the Chatmanager included in this plugin and the factions plugin.
Either the faction wouldn't show up, or the prefix wouldn't.
I posted a comment on the factions plugin page and this was the answer:
http://dev.bukkit.org/server-mods/factions/#c2618
Apparently the code wasn't updated to the new async chat events which were introduced in CraftBukkit 1.3.1, and you still use the old PlayerChatEvent.
if you updated that you would make a lot of people happy. you can't imagine how many people are having the same issue as me.

Edit:
Sorry I actually fixed this problem but I didn't post the solution here.
Basically you need to use essentials chat and set the config like this:
http://pastebin.com/DiQmRwnW
Remove Chatmanager and it should work.
Also remember to set these options:
http://pastebin.com/5Rc1XET3
It's only the fourth and the last option you have to look at.
